src/plugins/elasticsearch/lib/create_agent.js

```diff
@@ -19,7 +19,7 @@ module.exports = _.memoize(function (server) {
   }
 
   // Add client certificate and key if required by elasticsearch
-  if (config.get('elasticsearch.ssl.cert') && config.get('elasticsearch.ssl.key')) {
+  if (config.get('elasticsearch.ssl.enabled')) {
```

Comment:
Ah, good catch. This isn't related to the other changes at all. Search & replace gone bad